hw/ip/gpio/dv/cov/gpio_cov_excl.el

```diff
@@ -1,34 +1,55 @@
-// Copyright lowRISC contributors (OpenTitan project).
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
-// SPDX-License-Identifier: Apache-2.0
 //==================================================
 // This file contains the Excluded objects
-// Generated By User: gaurangg
+// Generated By User: marcelo
 // Format Version: 2
-// Date: Wed Oct 30 17:22:12 2019
+// Date: Fri Feb 14 17:12:55 2025
 // ExclMode: default
 //==================================================
-CHECKSUM: "3109872245 3727953184"
-INSTANCE: tb.dut.u_reg.u_data_in
-ANNOTATION: "[UNR]"
-Block 1 "0" "assign wr_en = de;"
-CHECKSUM: "1352181506 3803838024"
+CHECKSUM: "1706182284 132761700"
+INSTANCE: tb.dut.u_reg.u_reg_if.u_rsp_intg_gen
+ANNOTATION: "The parameter EnableRspIntgGen is zero, so the condition will never hit."
+Block 1 "461445014" "assign rsp_intg = tl_i.d_user.rsp_intg;"
+ANNOTATION: "The parameter EnableRspIntgGen is zero, so the condition will never hit."
+Block 2 "2643129081" "assign data_intg = tl_i.d_user.data_intg;"
```

Comment:
The annotation points at the wrong parameter, which is a little dubious...

But I think the same thing as I said about the previous item (and this can be in a parallel PR). Better to wire it to zero and add an assertion.

Reply:
I think there's a proper discussion about this [here](https://github.com/lowRISC/opentitan/pull/25767#discussion_r1963830459). Let's sort things out in one place, so I'll mark this item resolved.